server.js

```diff
@@ -1,4 +1,4 @@
-var secureServer = !!process.env.HTTPS;
+﻿var secureServer = !!process.env.HTTPS;
```

Comment:
That is because the line is indeed different. You saved the file as UTF-8 with a byte order mark (U+FEFF).